### PR TITLE
chore: publish packages with the @talend scope

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
The packages are published on npm with different ownerships.

**What is the chosen solution to this problem?**
We now have an npm organisation. Let's publish the packages with `@talend` scope.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[x] This PR introduces a breaking change**
The modules names changed.
bootstrap-talend-theme --> @talend/bootstrap-theme
talend-icons --> @talend/icons
talend-log --> @talend/log
react-cmf --> @talend/react-cmf
react-talend-components --> @talend/react-components
react-talend-containers --> @talend/react-containers
react-talend-forms --> @talend/react-forms

A script/codemod is available in `scripts/talend-ui-imports/replace-imports.sh`. Just execute 
```
./replace-imports.sh <your_frontend_project_path>
```

Tested on TDP OS only

### CODE REVIEW

File: scripts/talend-ui-imports/replace-here.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/cmf
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/components
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/containers
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/forms
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/icons
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/logging
+./replace-imports.sh /home/jsomsanith/Documents/workspace/ui/packages/theme
```
Comment (talend-glorieux) [inline scripts/talend-ui-imports/replace-here.sh]: You should remove that file.

File: scripts/talend-ui-imports/replace-imports.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+REPLACE_FORMS_SCHEMA='s/react-talend-forms/@talend\/react-forms/g'
+ALL_REPLACE="$REPLACE_THEME_SCHEMA; $REPLACE_ICON_SCHEMA; $REPLACE_LOG_SCHEMA; $REPLACE_CMF_SCHEMA; $REPLACE_COMPONENTS_SCHEMA; $REPLACE_CONTAINERS_SCHEMA; $REPLACE_FORMS_SCHEMA"
+sed -i "$ALL_REPLACE" $1/package.json
+
+# replace webpack sass data config
+print_title "Replace references in configurations (webpack)"
+REPLACE_TALEND_MODULES='s/react-talend-/@talend\\\//g'
+ALL_REPLACE_WITH_WILDCARD="$ALL_REPLACE; $REPLACE_TALEND_MODULES"
+sed -i "$ALL_REPLACE_WITH_WILDCARD" $1/*.js
+sed -i "$ALL_REPLACE_WITH_WILDCARD" $1/config/*.js
```
Comment (talend-glorieux) [inline scripts/talend-ui-imports/replace-imports.sh]: Why is there a code mode and that script?

File: scripts/utils.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+print_title () {
+  echo "----------------------------------------------------------------------"
+  echo $1
+  echo "----------------------------------------------------------------------"
+}
```
Comment (talend-glorieux) [inline scripts/utils.sh]: Not sure we need a file for that.

Comment (jsomsanith) [inline scripts/utils.sh]: For furthur scripts, but it's ok for me to move that in the main script for now

File: scripts/talend-ui-imports/replace-imports.sh [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+jscodeshift -t "${BASEDIR}/codemod.js" $APP_DIR/config
+
+print_title "Execute codemod on $APP_DIR/*.js (for webpack config)"
+jscodeshift -t "${BASEDIR}/codemod.js" $APP_DIR/*.js
+
+# replace bootstrap-talend-theme in all scss
+print_title "Replace bootstrap-talend-theme in all scss"
+find $APP_DIR/src -iregex '.*\.scss' | xargs sed -i '' 's/bootstrap-talend-theme/@talend\/bootstrap-theme/g'
+
+# replace modules in package.json
```
Comment (frassinier) [inline scripts/talend-ui-imports/replace-imports.sh]: I would like to suggest some few changes

```bash

#!/bin/bash

BASEDIR=$(dirname "$0")
APP_DIR=$1

# From version 0.77.0, the packages are published in the @talend organisation.
# This scripts :
#   * install jscodeshift globally if needed
#   * replace js imports in all the js files in src/, /examples, /stories
#   * replace imports in scss
#   * replace imports in src/**/*js
#   * replace references in configurations /*.js and /config/*.js
#   * replace references in package.json

print_title () {
  echo "----------------------------------------------------------------------"
  echo $1
  echo "----------------------------------------------------------------------"
}

print_success () {
  echo " ✓ done"
}

# [...] codeshift instructions go here

REPLACE_THEME_SCHEMA='s/bootstrap-talend-theme/@talend\/bootstrap-theme/g'
REPLACE_ICON_SCHEMA='s/talend-icons/@talend\/icons/g'
REPLACE_LOG_SCHEMA='s/talend-log/@talend\/log/g'
REPLACE_CMF_SCHEMA='s/react-cmf/@talend\/react-cmf/'
REPLACE_COMPONENTS_SCHEMA='s/react-talend-components/@talend\/react-components/g'
REPLACE_CONTAINERS_SCHEMA='s/react-talend-containers/@talend\/react-containers/g'
REPLACE_FORMS_SCHEMA='s/react-talend-forms/@talend\/react-forms/g'
ALL_REPLACE="$REPLACE_THEME_SCHEMA; $REPLACE_ICON_SCHEMA; $REPLACE_LOG_SCHEMA; $REPLACE_CMF_SCHEMA; $REPLACE_COMPONENTS_SCHEMA; $REPLACE_CONTAINERS_SCHEMA; $REPLACE_FORMS_SCHEMA"

# replace modules names in all scss files
print_title "Replace bootstrap-talend-theme in all scss"
find $APP_DIR/src -iregex '.*\.scss' -exec sed -i "$ALL_REPLACE" {} ";"
print_success

# replace modules names in package.json
print_title "Replace dependencies in package.json"
sed -i "$ALL_REPLACE" $APP_DIR/package.json
print_success
```

**Pull request merged**